README.md

```diff
@@ -76,7 +76,7 @@ curl http://0.0.0.0:4443/storage/v1/b/sample-bucket/o
 
 ## Client library examples
 
-For examples using the Python, Node.js and Go clients, check out the
+For examples using the Python, Node.js, Java and Go clients, check out the
```

Comment:
Oh man this is outdated, can you include C# and Scala here?

Reply:
Or you can leave it alone and I'll generalize it so we don't have to remember to always add languages to the readme.

Reply:
I would leave it as is.
Yep, I would recommend to have here a reference to the examples folder.